src/index.scss

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap');
+
+body {
+  margin: 0;
+  /* font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes. I thought I should leave it there for now and even maybe un-comment it so that it's a fallback in case something goes wrong with the fonts I imported. What do you think?

Reply:
I removed the commented out code.